src/main/java/io/aiven/kafka/connect/http/config/HttpSinkConfig.java

```diff
@@ -434,6 +450,13 @@ public final Long kafkaRetryBackoffMs() {
         return getLong(KAFKA_RETRY_BACKOFF_MS_CONFIG);
     }
 
+    public Map<String, String> getAdditionalHeaders() {
+        return getList(HTTP_HEADERS_ADDITIONAL).stream()
+                .map(s -> s.split(HTTP_HEADERS_ADDITIONAL_DELIMITER))
+                .filter(h -> h.length > 1)
```

Comment:
if we use the validator approach, then this line won't be needed in here anymore.

Reply:
Removed line